yarn-project/noir-bb-bench/src/index.ts

```diff
@@ -70,7 +70,7 @@ export async function proveCircuit1(
     const proverOutput = await backend.generateProof(witness);
     logger(`done generating recursive proof artifacts.`);
     return {
-      proof: proverOutput.proof as FixedLengthArray<string, 459>,
+      proof: proverOutput.proof as unknown as FixedLengthArray<string, 459>,
```

Comment:
Unrelated, but caused building with tsc to fail

Reply:
Seems like types were just wrong. These tests also don't seem to be run anywhere...